java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/metrics/ConnectorNodeMetrics.java

```diff
@@ -142,16 +144,16 @@ public static void incActiveSinkConnections(String sinkType, String ip) {
         activeSinkConnections.labels(sinkType, ip).inc();
     }
 
-    public static void decActiveConnections(String sinkType, String ip) {
+    public static void decActiveSinkConnections(String sinkType, String ip) {
         activeSinkConnections.remove(sinkType, ip);
     }
 
     public static void incSourceRowsReceived(String sourceType, String sourceId, double amt) {
         sourceRowsReceived.labels(sourceType, sourceId).inc(amt);
     }
 
-    public static void incSinkRowsReceived() {
-        sinkRowsReceived.inc();
+    public static void incSinkRowsReceived(String sinkType, String sinkId, double amt) {
+        sinkRowsReceived.labels(sinkType, sinkId).inc(amt);
```

Comment:
Just curious: why do we use `double` for `amt` here?

Reply:
BTW `sinkType` should be renamed to `connectorType` here as well.

Reply:
> Just curious: why do we use `double` for `amt` here?

because the `inc` method on the prometheus `Counter` takes a `double`..